internal/addon/rules/serviceNamesEndWithRule_test.go

```diff
@@ -0,0 +1,78 @@
+package rules_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/yoheimuta/go-protoparser/parser"
+	"github.com/yoheimuta/go-protoparser/parser/meta"
+	"github.com/yoheimuta/protolint/internal/addon/rules"
+	"github.com/yoheimuta/protolint/internal/linter/report"
+)
+
+func TestValidServiceNamesEndWithRule_Apply(t *testing.T) {
+	validTestCase := struct {
+		name         string
+		inputProto   *parser.Proto
+		wantFailures []report.Failure
+	}{
+		name: "no failures for proto with valid service names",
+		inputProto: &parser.Proto{
+			ProtoBody: []parser.Visitee{
+				&parser.Service{
+					ServiceName: "SomeServiceService",
+				},
+				&parser.Service{
+					ServiceName: "AnotherService",
+				},
+			},
+		},
+	}
+
+	t.Run(validTestCase.name, func(t *testing.T) {
+		rule := rules.NewServiceNamesEndWithRule("")
```

Comment:
[nits] I think you should set the argument to "Service" instead of empty. How about it?